code/modules/events/stray_cargo.dm

```diff
@@ -85,3 +86,4 @@
 
 /datum/round_event/stray_cargo/syndicate
 	possible_pack_types = list(/datum/supply_pack/misc/syndicate)
+	pod_type = /obj/structure/closet/supplypod/syndicate
```

Comment:
You shouldn't create a new supplypod subtype - we already have a system to handle this. Just call setStyle(STYLE_SYNDICATE) on an initialized supplypod object. That function will handle icons, descriptions, etc.
```suggestion
	pod = new /obj/structure/closet/supplypod()
	pod.setStyle(STYLE_SYNDICATE)
```

Reply:
You can find the styles you can use, etc. in the cargo.dm defines file

Reply:
thanks